felles/util/src/main/java/no/nav/vedtak/util/InputValideringRegex.java

```diff
@@ -63,6 +63,14 @@ public class InputValideringRegex {
      */
     public static final String BASE64_RFC4648_URLSAFE_WITH_PADDING = REGEXP_START + ALFABET_ENGELSK + TALL + "\\-_=" + REGEXP_SLUTT;
 
+
+    private static final String ORGNR = "^\\d{13}$";
+    private static final String AKTØR_ID = "^\\d{9}$";
```

Comment:
Bytte om de to? Legge til FNR med 11?

Reply:
Arbeidsgiver er enten ORGNR eller AKTØR, hhv 9 eller 13 siffer. Men arbeidsgiver er ikke FNR (utenom inputdata fra aareg/inntekt/im).

Reply:
Fnr fra selvbetjening også..
Men vet ikke om vi best holder fnr unna denne eller ikke